src/positiondetector/SimpleThreshold.cpp

```diff
@@ -178,9 +194,21 @@ void SimpleThreshold::tune(cv::Mat &frame, const oat::Position2D &position)
 
 void SimpleThreshold::applyThreshold(cv::Mat &frame)
 {
+
+    cv::inRange(frame,
+                1,
+                255,
+                nonmasked_frame_);
+
+    double mincomp_brightness =0;
+    if (mincomp_on_)
```

Comment:
Get rid of mincomp_on variable and just do mincomp_ != 0 here. 

Where is the value of mincomp_ being used? It looks like you are just always adding whatever the minimum of the non zero part of the frame to the threshold range.